vapixEvents/main.js

```diff
@@ -2,9 +2,9 @@ const { CameraVapix } = require('camstreamerlib/CameraVapix');
 
 var cv = new CameraVapix({
     'protocol': 'http',
-    'ip': '127.0.0.1',
+    'ip': '192.168.91.142',
```

Comment:
why do you use 192.168.91.142  and pass2?

Reply:
It was committed by mistake during the last amend.

Reply:
fixed